www/index.js

```diff
@@ -1654,6 +1654,12 @@ function update_session() {
         user_name = null;
         break;
       }
+      case "google_oauth_registering": {
```

Comment:
This is google_oauth_registering again :)
Also I would collapse this into a single case, similarly to unknown and google_oauth_registering above.

Reply:
Whoops, done, thanks!